OTOnexus.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'OTOnexus'
-  s.version          = '1.2.0'
+  s.version          = '1.3.0'
```

Comment:
I don't know if we have to keep bumping this by point version while we are still in development

Reply:
fair enough!